lms/static/sass/discussion/views/_home.scss

```diff
@@ -0,0 +1,166 @@
+// discussion - home layout
+// ====================
+
```

Comment:
Any need for a table of contents + sections here to match what you're doing in ``/_response.scss``?